### PR TITLE
Added sweet-lassi recipe

### PR DESCRIPTION
**Pull request by sayalitandel**
Hey @alfg! Please check my contribution. I've participated in the hacktoberfest for the first time. I would be glad if you would merge the PR and label it as hacktoberfest-accepted. Thank you!

### CODE REVIEW

File: src/recipes/sweet-lassi.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "name": "Lassi",
```
Comment (alfg) [inline src/recipes/sweet-lassi.json:2]:
```suggestion
    "name": "Sweet Lassi",
```

File: src/recipes/sweet-lassi.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            "measure": "pinch",
+            "ingredient": "Salt"
+        }
+    ],
+    "directions": [
+        "In a blender, blend the yogurt, ice, water, sugar and salt until mixture becomes frothy.",
+        "Pour mixture over ice cubes in tall glasses.",
+        "Now, relish it! :D"
+    ],
+    "image": "<sweet-lassi.jpg>",
```
Comment (alfg) [inline src/recipes/sweet-lassi.json:37]:
```suggestion
    "image": "sweet-lassi.jpg",
```

**Review by alfg — CHANGES_REQUESTED** [review]
Couple of suggestions. Thanks.

**Comment by sayalitandel** [comment]
Hey @alfg! I have made suggested changes. Please review it. I would be glad if you would merge the PR and label it as hacktoberfest-accepted. Thank you!

**Comment by alfg** [comment]
Thanks for fixing!

**Pull request merged**